integrations/serverless-ossm-traffic-isolation.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="serverless-ossm-traffic-isolation"]
+= Using {SMProductShortName} to isolate network traffic with {ServerlessProductName}
+:context: serverless-ossm-setup
+
+toc::[]
+
+:FeatureName: Using {SMProductShortName} to isolate network traffic with {ServerlessProductName}
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+{SMProductShortName} can be used to isolate network traffic between tenants on a shared OpenShift cluster using Service Mesh `AuthorizationPolicy` resources. {ServerlessProductShortName} can also leverage this, using several {SMProductShortName} resources. A tenant is a group of one or multiple OpenShift projects that can access each other over the network on a shared OpenShift cluster.
+
+[id="prerequisites_serverless-ossm-traffic-isolation"]
+== Prerequisites
+
+You have:
+
+* access to an {product-title} account with cluster administrator access
+
+* set up the {SMProductShortName} and {ServerlessProductShortName} integration
+
+* created one or more OpenShift projects for each tenant
```

Comment:
I realize this was changed during peer review, but I wanted to highlight for consideration that the IBM SG does have a section about lead-in sentences that states "Do not continue the lead-in sentence after the list".